api/tests/Behat/features-v2/reporting/sections/client-benefits-check/client-benefits-check.feature

```diff
@@ -0,0 +1,52 @@
+@client_benefits_check @v2 @v2_reporting_1
+Feature: Client benefits check
+
+    @lay-combined-high-not-started
+    Scenario: A deputy has checked the clients benefit entitlement on a specific date
+        Given a Lay Deputy has not started a Combined High Assets report
+        When I navigate to and start the client benefits check report section
+        And I confirm I checked the clients benefit entitlement on '01/01/2021'
+#        And I confirm others receive income on the clients behalf
```

Comment:
Can this be removed permanently or would they just be added back in with DDPB-4004?